### PR TITLE
Feature #294 - Send an email message with issue request to the website's support

### LINKED ISSUE #294: SP3: Send an email message with issue request to the website's support.

### PR DESCRIPTION
**Pull request by YaroslavBorysko**

### CODE REVIEW

File: beauty/api/views/contact_views.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    def post(self, request, *args, **kwargs):
+        """This method is used to send users questions to website's support."""
+        serializer = ContactFormSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        subject = "Beauty Support"
+        req_template = render_to_string("email/support_request.html", request.data)
+        resp_template = render_to_string("email/support_request_confirmation.html", request.data)
+        try:
+            send_mail(subject, req_template, "beautyproject689@gmail.com",
```
Comment (Misha86) [inline beauty/api/views/contact_views.py]: Maybe better get "beautyproject689@gmail.com" from settings.py?

File: beauty/api/views/contact_views.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                      ["testbeautyproject@gmail.com"], html_message=req_template)
+            send_mail(subject, resp_template, "beautyproject689@gmail.com",
```
Comment (Misha86) [inline beauty/api/views/contact_views.py]: Maybe better get "[beautyproject689@gmail.com](mailto:beautyproject689@gmail.com)" from settings.py?

Comment (Misha86) [reply beauty/api/views/contact_views.py]: settings.EMAIL_HOST_USER

**Review by Misha86 — CHANGES_REQUESTED** [review]
Use settings.EMAIL_HOST_USER to get email because when the admin changes email address will need to change it in your class too.

**Review by Misha86 — APPROVED** [review]

**Review by pazuzu-ua — APPROVED** [review]

**Review by shv833 — APPROVED** [review]

**Pull request merged**